libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -574,15 +574,15 @@ const AP_Param::GroupInfo NavEKF2::var_info[] = {
 
     // @Param: GSF_RUN_MASK
     // @DisplayName: Bitmask of which EKF-GSF yaw estimators run
-    // @Description: 1 byte bitmap of which EKF2 instances run an independant EKF-GSF yaw estimator to provide a backup yaw estimate that doesn't rely on magnetometer data. This estimator uses IMU, GPS and, if available, airspeed data. EKF-GSF yaw estimator data for the primary EKF2 instance will be logged as GSF0 and GSF1 messages. Use of the yaw estimate generated by this algorithm is controlled by the EK2_GSF_USE, EK2_GSF_DELAY and EK2_GSF_MAXCOUNT parameters. To run the EKF-GSF yaw estimator in ride-along and logging only, set EK2_GSF_USE to 0. 
```

Comment:
we could potentially remove the "1 byte" portion of the description.  Very few users will know what a "byte" is and it's not really necessary for them to know this detail.